scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -71,8 +71,8 @@ task GetToolVersions {
     # GVS generally uses the smallest `alpine` version of the Google Cloud SDK as it suffices for most tasks, but
     # there are a handlful of tasks that require the larger GNU libc-based `slim`.
     String cloud_sdk_slim_docker = "gcr.io/google.com/cloudsdktool/cloud-sdk:435.0.0-slim"
-    String variants_docker = "us.gcr.io/broad-dsde-methods/variantstore:2023-09-27-alpine"
-    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_10_07"
+    String variants_docker = "us.gcr.io/broad-dsde-methods/variantstore:2023-10-11-alpine"
+    String gatk_docker = "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_10_11"
```

Comment:
just sanity checking that you and @gbggrant are not going to collide since we're still living in a hashless world until [this gets merged](https://github.com/broadinstitute/gatk/pull/8549)

Reply:
I pulled in ah_var_store again, saw george's change to both of those, redid both of mine, and then ran an integration test again afterwards.  We should be fine